__tests__/integration/core-api/handlers/transactions.test.ts

```diff
@@ -129,12 +129,15 @@ describe("API 2.0 - Transactions", () => {
                 timestamp: 0,
                 version: 1,
                 type: 0,
+                typeGroup: 1,
                 fee: "0",
                 amount: "300000000000000",
+                blockId: expect.anything(), // ? how is that blockId isn't constant
```

Comment:
Block IDs used to be `BigInt` but are now hex, that is why.

Reply:
I feel that this assertion isn't right. I added it as stub. Block id is changing from run to run and it's integer. This is transaction.blockId, should it be hex?